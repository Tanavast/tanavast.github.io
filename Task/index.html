<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="/task/style.css">
    <title>Test</title>
</head>
<body >
    <section class="input-section">
        <input type="text" id="input-area" placeholder="Напишите что-то">
        <input type="button" value="START" id="input-btn">
    </section>
    <!-- /.input-section -->
    <section class="clock-section">
        <h2>Текущее время +3 часа:</h2>
        <h1 id="clock">00:00:00</h1>
    </section>
    <!-- /.clock-section -->
    
    <script src="/task/script.js"></script>
</body>
</html>
