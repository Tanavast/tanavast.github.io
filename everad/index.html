<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Everad Test</title>
<link href="/everad/assets/css/vendors.a4388800a17a3e35803c.css" rel="stylesheet"><link href="/everad/assets/css/app.a4388800a17a3e35803c.css" rel="stylesheet"></head>
<script src="https://kit.fontawesome.com/058c537ef0.js"></script>
<body>
  <header class="header" style="background-image: url(/everad/assets/img/header/bg-header-min.png)">
    <div class="top-line">
      <div class="container">
        <div class="row justify-content-lg-start justify-content-md-between">
          <div class="logo col-3 offset-xl-2 offset-md-0 offset-sm-6 offset-4">
            <div class="logo-img"><img src="/everad/assets/img/header/logo/logo.png" alt="Logo"></div>
            <div class="logo-text">Микоцин</div>
          </div>
          <div class="boolits col-7">
            <div class="boolit boolit-first"><i class="fas fa-medal"></i><h6>№1 Для избавления <br/>от любого типа грибка</h6></div>
            <div class="boolit boolit-second"><i class="fas fa-user-shield"></i><h6>Рекомендован <br/>дерматологами  </h6></div>
            <div class="boolit boolit-third"><i class="fas fa-users"></i><h6>13000+ довольных <br/>клиентов за 2018 год</h6></div>
          </div>
        </div>
      </div>
    </div>
    <div class="main-content">
      <div class="container">
        <div class="row justify-content-md-center justify-content-center">
          <div class="complex col-md-4 col-6 offset-xl-4 offset-md-1 order-xl-1 order-md-2">
            <div class="main-text">
              <h1>Комплексное <br/>избавление <br/>от грибка —</h1>
              <h2><span class="dotted-danger">изнутри</span> и <span class="dotted-success">снаружи</span></h2>
            </div>
            <div class="advantages">
              <div class="advantage advantage_first">
                <div class="circle">2в1</div>
                <div class="advantage_text"><h6>Комплекс Микоцин <b>устраняет<br/>зуд, неприятный запах</b><br/>и возвращает эстетичный вид</h6></div>
                <div class="arrow"><img src="/everad/assets/img/header/arrow.png" alt=""></div>
              </div>
              <div class="advantage advantage_second">
                <div class="circle">1</div>
                <div class="advantage_text"><h6><span class="light-danger"><b>Таблетки</b></span> Микоцин<br/><b> уничтожают все виды грибка</b><br/>на ногтях и коже</h6></div>
              </div>
              <div class="advantage">
                <div class="circle">2</div>
                <div class="advantage_text"><div class="advantage_text"><h6><span class="light-success"><b>Гель</b></span> Микоцин<br/><b>восстанавливает кожу</b><br/>и создает на ней <b>защитный<br/>барьер</b></h6></div>
              </div>
            </div>
          </div>
        </div>
          <div class="form col-6 col-md-3 order-xl-2 order-md-1" style="background-image: url(/everad/assets/img/header/forma-bg.png)">
            <div class="form-topline"><i class="fas fa-truck"></i><h6>Бесплатно</h6></div>
            <h5 class="form_text-top">Успейте заказать<br/>со скидкой<br/><span>до 10.09.2018</span></h5>
            <div class="price">
              <div class="old-price"><p>старая цена</p><span>918₽</span></div>
              <div class="new-price"><p>новая цена</p><span>459₽</span></div>
            </div>
            <div class="input-area">
              <div class="textbox">
                <i class="far fa-user"></i>
                <input type="text" class="input" placeholder="Имя">
              </div>
              <div class="textbox">
                <i class="fas fa-phone-alt"></i>
                <input type="tel" class="input" placeholder="Телефон">
              </div>
            </div>
            <button class="buy-btn"><span>Заказать</span><br/>со скидкой</button>
            <div class="bottom-text">Осталось <span>10</span> упаковок<br/>со скидкой</div>
            <div class="form-footer"><i class="fas fa-shield-alt"></i><h6>Ваши данные защищены!</h6></div>
          </div>
        </div>
      </div>
    </div>
  </header>
  <!-- ./header -->
  <main class="main">
      <section class="danger-section">
          <div class="container">
              <div class="row justify-content-center">
                <div class="col-12">
                    <div class="logo-img logo-content"><img src="/everad/assets/img/header/logo/logo.png" alt="Logo"></div>
                </div>
              </div>
          </div>
            <!-- /.container -->
            <div class="danger-infection_text">
              <div class="container">
                <div class="row justify-content-center">
                  <div class="col-8">
                    <div class="infection-main_text">
                        <h1>Грибок - это опасная инфекция</h1>
                        <h3>которая угрожает вам и вашим близким, ведь с каждым днем грибок приводит к:</h3>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <!-- /.danger-infection_text -->
            <div class="danger-infection_examples">
              <div class="container">
                <div class="row justify-content-center">
                  <div class="col-12 col-md-4 infection_example">
                    <div class="example-img">
                      <div class="first-example example">
                        <img src="/everad/assets/img/first_section/danger_1-min.png" alt="">
                      </div>
                      <div class="example-img_text">
                        <p>Сложным<br/>формам<br/>аллергии</p>
                      </div>
                    </div>
                    <div class="infection-text">
                      <p>Может появится сильная реакция<br/>на на привычную пищу, одежду,<br/>средства гигиены<br/></p>
                    </div>
                  </div>
                  <div class="col-12 col-md-4 infection_example">
                    <div class="example-img">
                      <div class="second-example example">
                        <img src="/everad/assets/img/first_section/danger_2-min.png" alt="">
                      </div>
                      <div class="example-img_text">
                        <p>Потере ногтя<br/>и ампутации<br/>пальца</p>
                      </div>
                    </div>
                    <div class="infection-text">
                      <p>После полного разрушения ногтя<br/>грибок распространяется на палец,<br/>что приводит к его ампутации<br/></p>
                    </div>
                  </div>
                  <div class="col-12 col-md-4 infection_example">
                    <div class="example-img">
                      <div class="third-example example">
                        <img src="/everad/assets/img/first_section/danger_3-min.png" alt="">
                      </div>
                      <div class="example-img_text">
                        <p>Тяжелому<br/>поражению<br/>внутренних<br/>органов<br/></p>
                      </div>
                    </div>
                    <div class="infection-text">
                      <p>Может привести к повреждениям<br/>мозга, слепоте, отказу печени, почек<br/>и сердца</p>
                    </div>
                  </div>
                  <!-- /.infection_example -->
                </div>
                <!-- /.row -->
              </div>
              <!-- /.contaner -->
            </div>
            <!-- /.danger-infection_examples -->
            <div class="statistics">
              <div class="container">
                <div class="row justify-content-center">
                  <div class="col-12 statistics-text">
                    <p><span>Статистика:</span> каждые 60 секунд врачи диагностируют опасные поражения внутренних органов,<br/>вызванные грибком, у 1-го пациентa</p>
                  </div>
                </div>
                <!-- /.row -->
              </div>
              <!-- /.continer -->
            </div>
            <!-- /.statistics -->
      </section>
      <!-- /.danger-section -->
      <section class="section-advantages">
         <div class="container">
            <div class="row justify-content-center">
              <div class="col-12 logo-section">
                  <div class="logo-img logo-content"><img src="/everad/assets/img/header/logo/logo.png" alt="Logo"></div>
              </div>
            </div>
        </div>
        <div class="section-advantages_main-text-box">
          <div class="cotainer">
            <div class="row justify-content-center">
              <div class="col-10 section-advantages_main-text">
                  <h1>Почему большинство средств против грибка<br/>малоэффективны </h1>
              </div>
            </div>
          </div>
        </div>
        <!-- /.section-advantages_main-text -->
        <div class="section-advantages_cards">
          <div class="container">
            <div class="row justify-content-center justify-content-md-around">
              <div class="col-12 col-md-4 first_card card">
                    <div class="card-img card-img_first"><img src="/everad/assets/img/second_section/icon_1-min.png" alt=""></div>
                    <h3 class="card-title">Чудо-средства</h3>
                    <h6 class="card-subtitle">Обещают, что избавят от грибка меньше, чем за месяц, хотя
                        это невозможно. </h6>
                    <div class="arrow-border"></div>
                    <h3 class="but">Ho</h3>
                    <p class="card-text">Ноготь – это роговой слой кожи.
                        Он не имеет живых клеток. Часть ногтя, которая уже поражена грибком, не может выздороветь
                        до полного его отрастания. Единственный способ вылечить грибок – подавлять его распространение все время,
                        пока ноготь не отрастет заново.
                        Для этого требуется 4-6 месяцев.
                        </p>
              </div>
              <div class="col-md-4 col-12 second_card card">
                    <div class="card-img card-img_second"><img src="/everad/assets/img/second_section/icon_2-min.png" alt=""></div>
                    <h3 class="card-title">Наружные средства</h3>
                    <h6 class="card-subtitle">Способны устранить внешние проявления грибка.<br/><span>(кремы, лаки, пластыри)</span></h6>
                    <div class="arrow-border"></div>
                    <h3 class="but">Ho</h3>
                    <p class="card-text">Очаг заражения находится
                        под ногтевой пластиной, а наружные средства  не способны его достичь - они действуют только на на верхнем слое ногтя, поэтому не способны избавить от грибка. Для повышения их эффективности рекомендуется спиливать ноготь полностью,
                        но это больно, некрасиво
                        и совершенно не дает гарантий выздоровления.
                        </p>
              </div>
              <div class="col-md-4 col-12 third_card card">
                  <div class="card-img card-img_third"><img src="/everad/assets/img/second_section/icon_3-min.png" alt=""></div>
                    <h3 class="card-title">Внутренние средства</h3>
                    <h6 class="card-subtitle">Убивают грибковую инфекцию
                        в организме.<br/><span>(таблетки, саше, капли)</span></h6>
                    <div class="arrow-border"></div>
                    <h3 class="but">Ho</h3>
                    <p class="card-text">Могут подействовать только
                        при длительном использовании 
                        (6 - 8 месяцев). За это время внутренние средства в 93%
                        случаев наносят серьезный
                        ущерб печени, вызывают сыпь, отеки, диарею, рвоту. К тому же, каждое средство эффективно
                        только против одной группы
                        грибка из 4000 видов.
                        </p>
              </div>
            </div>
            <!-- /.row -->
          </div>
          <!-- /.container -->
        </div>
        <!-- /.section-advantages_cards -->
        <div class="advantage-conclusion">
          <div class="container">
            <div class="row"> 
              <div class="col-12 advantage-conclusion_box">
                  <div class="advantage-conclusion_title"><h3>Вывод:</h3></div>
                  <div class="advantage-conclusion_text">
                    <p>Даже если вы используете наружные и внутренние средства одновременно, это не гарантирует
                        избавления от грибка: они могут конфликтовать между собой по составу и блокировать действие
                        друг друга или быть неэффективными против вашего типа грибка.</p>
                  </div>
              </div>
            </div>
            <!-- /.row -->
          </div>
          <!-- /.container -->
        </div>
        <!-- /.advantage-conclusion -->
      </section>
      <!-- /.section-advantages -->
      <div class="section-third">
        <div class="container">
          <div class="row justify-content-center">
            <div class="col-12">
              <div class="logo-img logo-content"><img src="/everad/assets/img/header/logo/logo-big.png" alt="Logo"></div>
            </div>
          </div>
        </div>
        <div class="section-third_text-box">
          <div class="container">
            <div class="row justify-content-center">
              <div class="col-12">
                  <div class="section-third_text-title">
                      <h1>Почему таблетки <span>Микоцин</span> от грибка <br/>так эффективны?</h1>
                      <h3>Другие средства не дают полную защиту от грибковых инфекций и могут быть опасны</h3>
                  </div>
              </div>
            </div>
          </div>
        </div>
        <div class="section-third_cards">
            <div class="container">
              <div class="row justify-content-center justify-content-md-around">
                <div class="col-12 col-md-4 section-third_card card">
                  <div class="half-circle-border"></div>
                  <div class="section-third_card-img card-img"><img src="/everad/assets/img/third_section/efficiency_icon_1-min.png" alt=""></div>
                    <h3 class="card-title">Кремы и мази</h3>
                    <p class="card-text">Не лечат грибок, потому 
                      что не проникают в ногтевую пластину. Только временно избавляют от симптомов инфекций. Могут вызвать сильную аллергию 
                      и раздражения на коже.
                    </p>
                </div>
                 <div class="col-12 col-md-4 section-third_card card">
                  <div class="half-circle-border"></div>
                  <div class="section-third_card-img card-img"><img src="/everad/assets/img/third_section/efficiency_icon_2-min.png" alt=""></div>
                    <h3 class="card-title">Кремы и мази</h3>
                    <p class="card-text">Не лечат грибок, потому 
                      что не проникают в ногтевую пластину. Только временно избавляют от симптомов инфекций. Могут вызвать сильную аллергию 
                      и раздражения на коже.
                    </p>
                </div>
                 <div class="col-12 col-md-4 section-third_card card success">
                  <div class="half-circle-border border-success"></div>
                  <div class="section-third_card-img card-img"><img src="/everad/assets/img/third_section/efficiency_icon_3-min.png" alt=""></div>
                    <h3 class="card-title">Кремы и мази</h3>
                    <p class="card-text">Не лечат грибок, потому 
                      что не проникают в ногтевую пластину. Только временно избавляют от симптомов инфекций. Могут вызвать сильную аллергию 
                      и раздражения на коже.
                    </p>
                </div>
              </div>
              <!-- /.row -->
            </div>
            <!-- /.container -->
          </div>
          <!-- /.section-third_cards -->
          <div class="section-third_footer">
            <div class="container">
              <div class="row justify-content-center">
                <div class="col-10">
                  <div class="section-third_footer-text">
                    <h2>Зачем лечить сложнее, дольше и дороже, если есть <span>Микоцин</span>?</h2>
                  </div>
                  
                </div>
              </div>
              <!-- /.row -->
            </div>
            <!-- /.container -->
          </div>
          <!-- /.section-third_footer -->
      </div>
      <!-- /.section-third -->
      <div class="section-fourth">
        <div class="container">
          <div class="row justify-content-center">
              <div class="col-12 logo-section">
                  <div class="logo-img logo-content"><img src="/everad/assets/img/header/logo/logo.png" alt="Logo"></div>
              </div>
              <div class="section-fourth_title">
                <h1>Как применять <span>Микоцин</span></h1>
              </div>
          </div>
        </div>
        <!-- /.container -->
        <div class="section-fourth_content">
          <div class="container">
            <div class="row justify-content-center">
              <div class="col-lg-4 col-10 section-fourth_content-img">
                <img src="/everad/assets/img/fourth_section/product_img-min.png" alt="">
              </div>
              <div class="col-lg-4 col-md-6 col-sm-8">
                <div class="instruction-container">
                  <div class="instruction-img">
                    <img src="/everad/assets/img/fourth_section/icon_1-min.png" alt="">
                  </div>
                  <div class="instruction-text">
                    <div class="green-line"></div>
                    <h4>Принимать по 1 таблетке<br/>утром и вечером</h4>
                  </div>
                </div>
                <div class="instruction-container">
                  <div class="instruction-img">
                    <img src="/everad/assets/img/fourth_section/icon_2-min.png" alt="">
                  </div>
                  <div class="instruction-text">
                    <div class="green-line"></div>
                    <h4>Запивать стаканом воды</h4>
                  </div>
                </div>
                <div class="instruction-container">
                  <div class="instruction-img">
                    <img src="/everad/assets/img/fourth_section/icon_3-min.png" alt="">
                  </div>
                  <div class="instruction-text">
                    <div class="green-line"></div>
                    <h4>Курс - от 30 дней</h4>
                  </div>
                </div>
              </div>
            </div>
            <!-- /.row -->
          </div>
          <!-- /.container -->
        </div>
        <!-- /.section-fourth_content -->  
      </div>
      <!-- /.section-fourth -->
  </main>
  <!-- /.main -->
<script type="text/javascript" src="/everad/assets/js/vendors.a4388800a17a3e35803c.js"></script><script type="text/javascript" src="/everad/assets/js/app.a4388800a17a3e35803c.js"></script></body>
</html>
