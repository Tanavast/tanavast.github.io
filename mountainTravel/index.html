<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>MountainTravel</title>
<link href="/mountainTravel/assets/css/app.0d6198bfe05b66d5815d.css" rel="stylesheet"></head>
<body>
    <div class="nav-menu">
        <a href="#" id="menu-btn" class="menu-btn"><span></span></a>
        <div class="menu">
          <nav id="menu-list" class="menu-list">
              <a href="#">-</a>
              <a href="#">-</a>
              <a href="#">-</a>
              <a href="#">-</a>
              <a href="#">-</a>
              <a href="#">-</a>
              <a href="#">-</a>
              <a href="#">-</a>
          </nav>
        </div>
      </div>

<div class="content">
    <div class="parallax-background">
        <div data-speed="15" class="layer" style="background: url(assets/img/parallaxBG/parallax_bg.jpg) center no-repeat; background-size: cover;"></div>
        <div data-speed="35" class="layer" style="background: url(assets/img/parallaxBG/parallax_layer1.png) center no-repeat; background-size: cover;"></div>
        <div data-speed="70" class="layer" style="background: url(assets/img/parallaxBG/parallax_layer2.png) center no-repeat; background-size: cover;"></div>
      </div>
      
      <header>
        <div class="top-line">
            <img src="assets/img/Лого.PNG" alt="logo" class="logo">
            <div class="phone"><i class="fas fa-mobile-alt"></i><div class="phone-number">+38(063)122-37-43</div></div>

        </div>
        
      </header>
      
      <div class="wrapper"></div>
      
      <footer></footer>
</div>


<script type="text/javascript" src="/mountainTravel/assets/js/vendors.0d6198bfe05b66d5815d.js"></script><script type="text/javascript" src="/assets/js/app.0d6198bfe05b66d5815d.js"></script></body>
</html>