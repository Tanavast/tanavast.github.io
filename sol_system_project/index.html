<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
    <link rel="stylesheet" href="src/css/main.css">
    <link rel="stylesheet" href="src/css/content.css">
    <link rel="stylesheet" href="src/css/rotate_section.css">
    <link rel="stylesheet" href="src/css/main_shadow.css">
    <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Black+Ops+One">
</head>
<body>
    <header class="header">
        <div class="header_content">
            <h1>Solar system</h1>
            <h4>View all planets of the solar system</h4>
        </div>
    </header>
    <div class="crutch">
        <div class="planet_viwer_headline">
            <h2>Planet viewer</h2>
            <h5>Select the planet</h5>
        </div>
        <main class="planet_viewer">
            <section class="section section_planets_left">
                <div class="left_left">
                        <div class="mercury planet_section planet_left_left_top">
                            <a href="" class="planet_name">Mercury</a>
                        </div>
                        <div class="venus planet_section planet_left_left_bottom">
                            <a href="" class="planet_name">Venus</a>
                        </div>
                </div>
                <div class="left_right">
                        <div class="earth planet_section planet_left_right_top">
                            <a href="" class="planet_name">Earth</a>
                        </div>
                        <div class="mars planet_section planet_left_right_bottom">
                            <a href="" class="planet_name">Mars</a>
                        </div>
                </div>
            </section>
            <section class="section section_planets_right">
                <div class="right_left">
                        <div class="jupiter planet_section planet_right_left_top">
                            <a href="" class="planet_name">Jupiter</a>
                        </div>
                        <div class="saturn planet_section planet_right_left_bottom">
                            <a href="" class="planet_name">Saturn</a>
                        </div>
                </div>
                <div class="right_right">
                        <div class="uranus planet_section planet_right_right_top">
                            <a href="" class="planet_name">Uranus</a>
                        </div>
                        <div class="neptune planet_section planet_right_right_bottom">
                            <a href="" class="planet_name">Neptune</a>
                        </div>
                </div>
            </section>
        </main>
        <footer class="footer">
            <h6>&copy Kuts Yaroslav 2019</h6>
        </footer>
    </div>
    <script src="/src/js/main.js"></script>
</body>
</html>