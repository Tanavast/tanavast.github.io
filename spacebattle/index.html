<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Space battle</title>
    <link rel="stylesheet" href="/spacebattle/css/initialScreen.css" type="text/css" charset="utf-8" />
    <link rel="stylesheet" href="/spacebattle/css/battlefield.css">
    <link href="https://fonts.googleapis.com/css?family=Black+Ops+One" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
    <link rel="stylesheet" href="/spacebattle/css/fonts.css" type="text/css" charset="utf-8" />
</head>
<body>
    <div class="initialScreen" id="initialScreen">
        <div class="background_doors">
            <div class="left-door door">
                <h1 class="gameName">space</h1>
            </div>
            <div class="right-door door">
                <div class="play-scoreboard"><div class="play-btn">8.8.8.8</div><div class="play-btn_active">p.l.a.y</div></div>
                <h1 class="gameName">battle</h1>
            </div>
        </div>
    </div>

    <div class="battlefield">
        <!-- верхний блок для вывода текстовых сообщений -->
        <div id="text_top" class="text-top">placement</div>
        <div class="outer clearfix">
            <!-- поле игрока -->
            <div class="fields">
                <div class="field field-user">
                    <div id="field_user" class="ships"></div>
                </div>
     
                <!-- поле компьютера -->
                <div class="field field-comp hidden">
                    <div id="field_comp" class="ships"></div>
                </div>
            </div>
     
            <!-- инструкция -->
            <div id="instruction" class="instruction">
                <div id="type_placement" class="type-placement-box">
                    1.<span class="link" data-target="random">Random</span></br>
                    2.<span class="link" data-target="manually">manually</span>
                </div>
                <div id="ships_collection" class="ships-collection hidden" > 
                    <p>Drag the ships on the playing field with your mouse. To set the ship vertically, click on it with the right mouse button.</p>
                    <!-- набор кораблей для перетаскивания -->
                    <ul id='initial_ships' class="initial-ships">
                        <li>
                            <div id="fourdeck1" class="ship fourdeck"></div>
                            <div id="tripledeck1" class="ship tripledeck tripledeck1"></div>
                            <div id="tripledeck2" class="ship tripledeck tripledeck2"></div>
                        </li>
                        <li>
                            <div id="doubledeck1" class="ship doubledeck"></div>
                            <div id="doubledeck2" class="ship doubledeck doubledeck2"></div>
                            <div id="doubledeck3" class="ship doubledeck doubledeck3"></div>
                        </li>
                        <li>
                            <div id="singledeck1" class="ship singledeck"></div>
                            <div id="singledeck2" class="ship singledeck singledeck2"></div>
                            <div id="singledeck3" class="ship singledeck singledeck3"></div>
                            <div id="singledeck4" class="ship singledeck singledeck4"></div>
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        <!-- кнопка запуска игры -->
        <span id="play" class="btn-play hidden">start game</span>
        <!-- нижний блок для вывода текстовых сообщений -->
        <div id="text_btm" class="text-btm hidden"></div>
    </div>
    <script src="/spacebattle/initialScreen.js"></script>
    <script src="/spacebattle/resources/shipPlacement.js"></script>
</body>
</html>