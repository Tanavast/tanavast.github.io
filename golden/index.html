<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
    <link rel="stylesheet" href="/src/css/preloader.css">
    <link rel="stylesheet" href="/src/css/main.css">
    <link rel="stylesheet" href="/src/css/header.css">
    <link rel="stylesheet" href="/src/css/main_content.css">
    <link rel="stylesheet" href="/src/css/services.css">
    <link rel="stylesheet" href="/src/css/portfolio.css">
    <link rel="stylesheet" href="/src/css/about.css">
</head>
<body>
    <div class="preloader" id="page_preloader">
        <div class="loader"></div>
    </div>
    <header class="nav">
        <div class="nav">
            <div class="logo">
                <img src="/src/resources/img/Logo-Treehouse.png" alt="logo">
            </div>
            <div class="nav_menu">
                <ul class="navbar">
                    <li class="nav_btn"><a href="#main">Home</a></li>
                    <li class="nav_btn"><a href="#services">Services</a></li>
                    <li class="nav_btn"><a href="#portfolio">Portfolio</a></li>
                    <li class="nav_btn"><a href="#about">About</a></li>
                </ul>
            </div>
        </div>
    </header> 
    <main class="main" id="main"> 
        <div class="greetings">
            <h2>welcome to our studio!</h2>
            <h1>It`s nice to meet you</h1>
        </div>
        <div class="more_btn">
            <a href="#">tell me more</a>
        </div>  
    </main>
    <div class="services" id="services">
        <h2 class="services_title">Services</h2>
        <h5 class="services_subtitle">Lorem ipsum dolor sit amet, consectetur adipisicing elit</h5>
        <div class="service_blocks">
            <div class="service">
                <img src="/src/resources/icons/icon1.png" alt="E-Commerce">
                <h4 class="service_title">E-Commerce</h4>
                <p class="service_description">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio, impedit modi! Quos minus incidunt veniam quas odio, nemo, debitis temporibus soluta aspernatur odit autem harum, error sed asperiores? Assumenda, cumque.</p>
            </div>
            <div class="service">
                <img src="/src/resources/icons/Icon-Laptop.png" alt="Responsive Web">
                <h4 class="service_title">Responsive Web</h4>
                <p class="service_description">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio, impedit modi! Quos minus incidunt veniam quas odio, nemo, debitis temporibus soluta aspernatur odit autem harum, error sed asperiores? Assumenda, cumque.</p></div>
            <div class="service">
                <img src="/src/resources/icons/Icon-Locked.png" alt="Web Security">
                <h4 class="service_title">Web Security</h4>
                <p class="service_description">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio, impedit modi! Quos minus incidunt veniam quas odio, nemo, debitis temporibus soluta aspernatur odit autem harum, error sed asperiores? Assumenda, cumque.</p></div>
        </div>
    </div>
    <div class="portfolio" id="portfolio">
            <h2 class="portfolio_title">Our portfolio</h2>
            <h5 class="portfolio_subtitle">Proin iaculis purus consequat sem cure.</h5>
            <div class="portfolio_blocks">
                <div class="portfolio_block example_one">
                </div>
                <div class="portfolio_block example_two">
                </div>
                <div class="portfolio_block example_three">
                </div>
            </div>
    </div>
    <div class="about" id="about">
            <h2 class="about_title">Abouts us</h2>
            <h5 class="about_subtitle">Proin iaculis purus consequat sem cure.</h5>
            <div class="about_block">
                <div class="about_text">
                    <h4 class="about_block_title">July 2010</h4>
                    <h5 class="about_block_subtitle">Our Humble Beginnings</h5>
                    <p class="about_block_description">Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt. Proin 
                            iaculis purus consequat sem cure digni 
                            ssim. Donec porttitora entum suscipit.</p>
                </div>
                <div class="about_icon about_icon_one"></div>
            </div>
            <div class="about_block">
                    <div class="about_icon about_icon_two"></div>
                    <div class="about_text">
                        <h4 class="about_block_title">January 2011</h4>
                        <h5 class="about_block_subtitle">Facing Startup Battles</h5>
                        <p class="about_block_description">Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt. Proin 
                                iaculis purus consequat sem cure digni 
                                ssim. Donec porttitora entum suscipit.</p>
                    </div>
            </div>
            <div class="about_block">
                    <div class="about_text">
                        <h4 class="about_block_title">December 2012</h4>
                        <h5 class="about_block_subtitle">Enter The Dark Days</h5>
                        <p class="about_block_description">Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt. Proin 
                            iaculis purus consequat sem cure digni 
                            ssim. Donec porttitora entum suscipit.</p>
                 </div>
                <div class="about_icon about_icon_three"></div>
            </div>
            <div class="about_block">
                <div class="about_icon about_icon_four"></div>
                <div class="about_text">
                        <h4 class="about_block_title">february 2014</h4>
                        <h5 class="about_block_subtitle">Our Triumph</h5>
                        <p class="about_block_description">Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt. Proin 
                            iaculis purus consequat sem cure digni 
                            ssim. Donec porttitora entum suscipit.</p>
                </div>
            </div>
    </div>
    <footer class="footer">
        <span>Â© Copyright 2014 FreebiesXpress.com</span>
    </footer>
    <script src="/src/libraries/Jquery/jquery.js"></script>
    <script src="/src/js/onscroll.js"></script>
    <script src="/src/js/preloader.js"></script>
</body>
</html>